### DISCUSSION TITLE
Error: The network has multiple inputs, and saving such result han't been implemented.

### DISCUSSION THREAD

**Discussion by MustafaHull** [lululxvi/deepxde#1734] [General]
I am testing the simple example of antiderivative DeepONet, the run was successful up to the training stage, but after that, two-error come out:

# Parameters: 7401
Saving loss history to C:\Users\Dell\OneDrive - hull.ac.uk\Wave Eneergy Work\deeponet-master\src\loss.dat ...
Error: The network has multiple inputs, and saving such result han't been implemented.
Error: The network has multiple inputs, and plotting such result han't been implemented.

the full output is as below:
================================== RESTART: C:\Users\Dell\OneDrive - hull.ac.uk\Wave Eneergy Work\deeponet-master\src\deeponet_pde.py =================================
WARNING:tensorflow:From C:\Users\Dell\AppData\Local\Programs\Python\Python39\lib\site-packages\keras\src\losses.py:2976: The name tf.losses.sparse_softmax_cross_entropy is deprecated. Please use tf.compat.v1.losses.sparse_softmax_cross_entropy instead.

Using backend: tensorflow.compat.v1
Other supported backends: tensorflow, pytorch, jax, paddle.
paddle supports more examples now and is recommended.
WARNING:tensorflow:From C:\Users\Dell\AppData\Local\Programs\Python\Python39\lib\site-packages\deepxde\backend\tensorflow_compat_v1\tensor.py:25: The name tf.disable_v2_behavior is deprecated. Please use tf.compat.v1.disable_v2_behavior instead.

WARNING:tensorflow:From C:\Users\Dell\AppData\Local\Programs\Python\Python39\lib\site-packages\tensorflow\python\compat\v2_compat.py:108: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
Generating operator data...
'gen_operator_data' took 175.005780 s

Generating operator data...
'gen_operator_data' took 1317.193413 s

Compiling model...
Building DeepONet...

Warning (from warnings module):
  File "C:\Users\Dell\AppData\Local\Programs\Python\Python39\lib\site-packages\deepxde\nn\tensorflow_compat_v1\deeponet.py", line 367
    return tf.layers.dense(
UserWarning: `tf.layers.dense` is deprecated and will be removed in a future version. Please use `tf.keras.layers.Dense` instead.
WARNING:tensorflow:From C:\Users\Dell\AppData\Local\Programs\Python\Python39\lib\site-packages\keras\src\utils\version_utils.py:76: The name tf.executing_eagerly_outside_functions is deprecated. Please use tf.compat.v1.executing_eagerly_outside_functions instead.

'build' took 5.074972 s

WARNING:tensorflow:From C:\Users\Dell\AppData\Local\Programs\Python\Python39\lib\site-packages\deepxde\model.py:168: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.

'compile' took 6.762535 s

Warning: epochs is deprecated and will be removed in a future version. Use iterations instead.
Training model...

Step      Train loss    Test loss     Test metric   
0         [3.76e-01]    [3.60e-01]    [3.52e-01]    
1000      [4.29e-04]    [4.69e-04]    [4.56e-04]    
2000      [1.25e-04]    [1.39e-04]    [1.34e-04]    
3000      [1.00e-04]    [1.13e-04]    [1.09e-04]    
4000      [8.92e-05]    [1.01e-04]    [9.80e-05]    
5000      [5.04e-05]    [5.87e-05]    [5.63e-05]    
6000      [3.15e-05]    [3.60e-05]    [3.46e-05]    
7000      [4.05e-05]    [4.37e-05]    [4.25e-05]    
8000      [1.95e-05]    [2.28e-05]    [2.20e-05]    
9000      [1.47e-05]    [1.74e-05]    [1.66e-05]    
10000     [1.75e-05]    [1.92e-05]    [1.86e-05]    
11000     [9.42e-06]    [1.08e-05]    [1.03e-05]    
12000     [1.26e-05]    [1.42e-05]    [1.37e-05]    
13000     [9.21e-06]    [1.03e-05]    [9.92e-06]    
14000     [7.55e-06]    [8.85e-06]    [8.50e-06]    
15000     [6.43e-06]    [7.31e-06]    [6.99e-06]    
16000     [5.47e-06]    [6.34e-06]    [6.03e-06]    
17000     [1.16e-05]    [1.23e-05]    [1.19e-05]    
18000     [5.36e-06]    [5.94e-06]    [5.66e-06]    
19000     [1.50e-05]    [1.59e-05]    [1.54e-05]    
20000     [1.31e-05]    [1.37e-05]    [1.33e-05]    
21000     [6.99e-06]    [7.27e-06]    [6.98e-06]    
22000     [4.51e-06]    [5.00e-06]    [4.75e-06]    
23000     [5.36e-06]    [5.76e-06]    [5.50e-06]    
24000     [3.44e-06]    [3.88e-06]    [3.65e-06]    
25000     [3.16e-06]    [3.58e-06]    [3.36e-06]    
26000     [2.91e-06]    [3.32e-06]    [3.10e-06]    
27000     [5.88e-06]    [6.15e-06]    [5.90e-06]    
28000     [2.64e-06]    [3.01e-06]    [2.80e-06]    
29000     [3.09e-06]    [3.39e-06]    [3.17e-06]    
30000     [4.71e-06]    [5.00e-06]    [4.75e-06]    
31000     [4.15e-06]    [4.48e-06]    [4.24e-06]    
32000     [4.32e-06]    [4.51e-06]    [4.29e-06]    
33000     [8.34e-06]    [8.66e-06]    [8.32e-06]    
34000     [6.72e-06]    [6.95e-06]    [6.67e-06]    
35000     [2.59e-06]    [2.92e-06]    [2.70e-06]    
36000     [3.08e-06]    [3.39e-06]    [3.18e-06]    
37000     [6.25e-06]    [6.55e-06]    [6.27e-06]    
38000     [3.51e-06]    [3.87e-06]    [3.64e-06]    
39000     [9.01e-06]    [8.83e-06]    [8.56e-06]    
40000     [5.93e-06]    [6.09e-06]    [5.84e-06]    
41000     [2.43e-06]    [2.73e-06]    [2.53e-06]    
42000     [2.13e-06]    [2.38e-06]    [2.17e-06]    
43000     [2.82e-06]    [3.08e-06]    [2.87e-06]    
44000     [2.48e-06]    [2.82e-06]    [2.60e-06]    
45000     [1.81e-06]    [2.11e-06]    [1.90e-06]    
46000     [1.91e-06]    [2.19e-06]    [1.99e-06]    
47000     [1.75e-06]    [2.05e-06]    [1.84e-06]    
48000     [5.09e-06]    [5.44e-06]    [5.17e-06]    
49000     [8.45e-06]    [8.69e-06]    [8.39e-06]    
50000     [2.91e-06]    [3.27e-06]    [3.04e-06]    

Best model at step 47000:
  train loss: 1.75e-06
  test loss: 2.05e-06
  test metric: [1.84e-06]

'train' took 1599.060037 s

# Parameters: 7401
Saving loss history to C:\Users\Dell\OneDrive - hull.ac.uk\Wave Eneergy Work\deeponet-master\src\loss.dat ...
Error: The network has multiple inputs, and saving such result han't been implemented.
Error: The network has multiple inputs, and plotting such result han't been implemented.

**Comment by xyy386** [comment]
I have met the same problem, have you solve it?

**Comment by lululxvi** [comment]
Don't use the saveplot function in DeepXDE.
